CHANGELOG.md

```diff
@@ -4,13 +4,9 @@
 
 [Full Changelog](https://github.com/jwt/ruby-jwt/compare/v2.9.0...main)
 
-**Features:**
-
-- Your contribution here
-
 **Fixes and enhancements:**
 
-- Your contribution here
+- Updated README to correctly document `OpenSSL::HMAC` documentation.
```

Comment:
The format would be something like this. Also keeping the placeholder would be cool.

```suggestion
- Updated README to correctly document `OpenSSL::HMAC` documentation [#617](https://github.com/jwt/ruby-jwt/pull/617) ([@aedryan](https://github.com/aedryan))
- Your contribution here
```